lib/zipsign/file.cc

```diff
@@ -67,6 +72,13 @@ size_t File::read(void * buffer, size_t count, bool check)
     return result;
 }
 
+#if defined(_WIN32)
```

Comment:
ftruncate should be moved to an anonymous namespace

Reply:
A anonymous namespace is used to keep symbols local for the current module. This is similar to static functions in C.
I may change this later.